Problem after updating to v2.2.0

Updated to 2.2.0 and getting the following error while running `sst dev`. Seen a few more people mentioning this on Discord with no discernible solution.

```
✖ Deploying bootstrap stack, this only needs to happen once
Error: Cannot find module 'archiver'
Require stack:
- /Users/danny/src/spatula-next/node_modules/sst-aws-cdk/lib/api/hotswap/lambda-functions.js
- /Users/danny/src/spatula-next/node_modules/sst-aws-cdk/lib/api/hotswap-deployments.js

Trace: Error: Cannot find module 'archiver'
Require stack:
- /Users/danny/src/spatula-next/node_modules/sst-aws-cdk/lib/api/hotswap/lambda-functions.js
- /Users/danny/src/spatula-next/node_modules/sst-aws-cdk/lib/api/hotswap-deployments.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:985:15)
    at Module._load (node:internal/modules/cjs/loader:833:27)
    at Module.require (node:internal/modules/cjs/loader:1051:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/Users/danny/src/spatula-next/node_modules/sst-aws-cdk/lib/api/hotswap/lambda-functions.ts:10:18)
    at Module._compile (node:internal/modules/cjs/loader:1149:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1203:10)
    at Module.load (node:internal/modules/cjs/loader:1027:32)
    at Module._load (node:internal/modules/cjs/loader:868:12)
    at Module.require (node:internal/modules/cjs/loader:1051:19)
    at process.<anonymous> (file:///Users/danny/src/spatula-next/node_modules/sst/cli/sst.js:56:17)
    at process.emit (node:events:525:35)
    at process.processEmit [as emit] (/Users/danny/src/spatula-next/node_modules/signal-exit/index.js:199:34)
    at process._fatalException (node:internal/process/execution:149:25)
    at processPromiseRejections (node:internal/process/promises:288:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)

Need help with this error? Join the SST community on Discord https://sst.dev/discord
```

--- a/.github/workflows/aws-cdk.yml
+++ b/.github/workflows/aws-cdk.yml
@@ -42,7 +42,7 @@ jobs:
           PACKAGE_JSON=./package/package.json
           VERSION=$(jq -r .version $PACKAGE_JSON)
           jq '
-            .dependencies += {chalk: .devDependencies.chalk, yaml: .devDependencies.yaml, promptly: .devDependencies.promptly} |
+            .dependencies += {chalk: .devDependencies.chalk, yaml: .devDependencies.yaml, promptly: .devDependencies.promptly, archiver: .devDependencies.archiver } |
             .name = "sst-aws-cdk"
           ' $PACKAGE_JSON > tmp.json
           mv tmp.json $PACKAGE_JSON

